src/main/scala/nl.knaw.dans.easy.solr4files/components/FileItem.scala

```diff
@@ -42,20 +40,18 @@ case class FileItem(bag: Bag, ddm: DDM, xml: Node) extends DebugEnhancedLogging
     // @formatter:off
   }
 
-  def shouldIndex: Boolean = {
+  val path: String = xml.attribute("filepath").map(_.text).getOrElse("")
+  val accessibleTo: String = ( xml \ "accessRights").map(_.text).mkString match {
+    case s if s.isEmpty => datasetAccessitbleTo
```

Comment:
1. can't you just match on `""` here?
2. `datasetAccessibleTo`? (without extra `t`)